pkg/services/requests.go

```diff
@@ -37,13 +43,34 @@ func GenerateRequestsTable(requests []clientapi.AccessRequestClientModel) *uitab
 			revocationTime = "NA"
 		}
 
-		table.AddRow(request.Id, utils.DisplayTime(creationTime), revocationTime, integrations, *request.Justification.Get(), coloredStatus(request.Status.Status))
+		table.AddRow(request.Id, utils.DisplayTime(creationTime), revocationTime, integrations, *request.Justification.Get(), ColoredStatus(request))
 	}
 
 	return table
 }
 
-func GetUserLastRequest(ctx context.Context, client *aponoapi.AponoClient) (*clientapi.AccessRequestClientModel, error) {
+func WaitForNewRequest(ctx context.Context, client *aponoapi.AponoClient, creationTime time.Time, timeout time.Duration) (*clientapi.AccessRequestClientModel, error) {
```

Comment:
why you moved it? looks like the only place you call it is from `request_loader/request.go`

Reply:
you are right, will return it

Reply:
sorry I conused you, this function was in the `new.go` file and was used there, I moved it to the `services/requests.go` file and now the loader is using it instead of the command it self (the command use the loader instead of this function).